tests/system/HTTP/ResponseCookieTest.php

```diff
@@ -255,6 +256,7 @@ public function testCookieStrictSetSameSite()
 
     public function testCookieBlankSetSameSite()
     {
+        /** @var \Config\Cookie $config */
```

Comment:
there is alias for it, so use `CookieConfig` for shorter:

```suggestion
        /** @var CookieConfig $config */
```

Reply:
Done.